deployment/CP05MOAS-A6264_Deploy.csv

```diff
@@ -1,71 +1,99 @@
 #AR-28,,AR-28,,CP05MOAS-A6264-06-PARADN000,5,1,2018-03-31T12:16:00,2018-03-31T12:52:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50266,39.91799,-70.76098,,,,mission aborted at 12:31 UTC due to ground fault (GFI); no data
-P10a,,P10a,,CP05MOAS-A6264-00-ENG000000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,39.9184,-70.8985,,,,across shelf surveys
-P10a,,P10a,,CP05MOAS-A6264-01-FLORTN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01646,39.9184,-70.8985,,,,across shelf surveys
-P10a,,P10a,,CP05MOAS-A6264-02-DOSTAN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-02909,39.9184,-70.8985,,,,across shelf surveys
-P10a,,P10a,,CP05MOAS-A6264-03-CTDAVN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00425,39.9184,-70.8985,,,,across shelf surveys
-P10a,,P10a,,CP05MOAS-A6264-04-NUTNRN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-01121,39.9184,-70.8985,,,,across shelf surveys
-P10a,,P10a,,CP05MOAS-A6264-05-ADCPAN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.9184,-70.8985,,,,across shelf surveys
-P10a,,P10a,,CP05MOAS-A6264-06-PARADN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50266,39.9184,-70.8985,,,,across shelf surveys
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-00-ENG000000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,39.9184,-70.8985,,,,across-shelf survey
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-01-FLORTN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01646,39.9184,-70.8985,,,,across-shelf survey
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-02-DOSTAN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-02909,39.9184,-70.8985,,,,across-shelf survey
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-03-CTDAVN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00425,39.9184,-70.8985,,,,across-shelf survey
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-04-NUTNRN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-01121,39.9184,-70.8985,,,,across-shelf survey
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-05-ADCPAN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.9184,-70.8985,,,,across-shelf survey
+WJ20180705,,WJ20180705,,CP05MOAS-A6264-06-PARADN000,6,1,2018-04-24T06:40:00,2018-04-24T16:40:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50266,39.9184,-70.8985,,,,across-shelf survey

```

Comment:
@cdobs These cruise numbers (WJ20180705) are suspect... They are after the deployment date of this AUV (2018-04-24).

Reply:
Updated to WJ20180423 which is consistent with the tracking sheet now